features/cloud/file.include/etc/ssh/ssh_config

```diff
@@ -6,8 +6,9 @@ HostbasedAuthentication no
 StrictHostKeyChecking no
 
 # Use secure ciphers
-Ciphers aes128-ctr,aes192-ctr,aes256-ctr
-MACs hmac-sha2-256,hmac-sha2-512
+Ciphers chacha20-poly1305@openssh.com,aes256-gcm@openssh.com,aes128-gcm@openssh.com,aes256-ctr,aes192-ctr,aes128-ctr
+MACs hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-512,hmac-sha2-256,umac-128@openssh.com
+KexAlgorithms curve25519-sha256@libssh.org,ecdh-sha2-nistp521,ecdh-sha2-nistp384,ecdh-sha2-nistp256,diffie-hellman-group-exchange-sha256
 
```

Comment:
I used the ciphers from the official hardening guide. The extra ones were not in the guide